Sources/SentrySwiftUI/Preview/SentryReplayMaskPreviewUIView.swift

```diff
@@ -0,0 +1,71 @@
+#if canImport(SwiftUI) && canImport(UIKit) && os(iOS) || os(tvOS)
+import Sentry
+import SwiftUI
+import UIKit
+
+#if CARTHAGE || SWIFT_PACKAGE
+@_implementationOnly import SentryInternal
+#endif
+
+class SentryReplayMaskPreviewUIView: UIView {
```

Comment:
`h`: We can move this to the main Sentry package as it has no dependencies on SwiftUI. @brustolin I think we could also use this for non SwiftUI apps. We could port this so people can use this with an option for any iOS app while debugging. With that we could also keep the `SentryViewPhotographer` internal.

Reply:
We could, not sure if we should. It think session replay should have its own lib.
Moving this to main just change what needs to be public, because this would need to be public in order for SentrySwiftUI to use it.

Lets go step-by-step.
After finishing this PR for SwiftUI we can debate debugging it for UIKit that has no preview.

Reply:
This is just a matter of deciding what we want to make public. SentryReplayMaskPreviewUIView should not be directly used by the user. There is a correct way to use it, but most users will likely find it uncomfortable or inconvenient.

This arbitrary decision about what should be public and what should not is only making our lives harder without providing any real benefit. Users only care about having code snippets they can copy and paste to make Sentry work.

Reply:
This is just a matter of deciding what we want to make public. SentryReplayMaskPreviewUIView should not be directly used by the user either. There is a correct way to use it and most users will not know how to correct use it.

I believe we're overthinking this "what should be public and what should not" thing, and this only makes our lives harder without providing any real benefit. Users only care about having code snippets they can copy and paste to make Sentry work.

Reply:
> I believe we're overthinking this "what should be public and what should not" thing, and this only makes our lives harder without providing any real benefit. Users only care about having code snippets they can copy and paste to make Sentry work.

If users can use it, they will, and then we will break them, which I would like to avoid.

Reply:
> If users can use it, they will

That's the part I believe we're overthinking.

I will open another PR to start the UIKit debug feature and reorganize the public code.